tests/test_pcdMap.py

```diff
@@ -11,7 +11,7 @@
 class TestPcdMap(MappingTestCase):
 
     def test_PcdMap(self):
-        coeff = 0.2
+        coeff = 0.002
```

Comment:
Because the larger distortion was unrealistically large -- sufficiently so that the inverse could not be computed to reasonable precision. I had to weaken it in order to test the inverse.